test/common/index.js

```diff
@@ -804,3 +804,6 @@ exports.runWithInvalidFD = function(func) {
 
   exports.printSkipMessage('Could not generate an invalid fd');
 };
+
+const utilDebugLog = util.debuglog('test');
```

Comment:
If the intention is to get all the tests to slowly refactor all the tests to `common.debuglog`, then I am -1 to this since  I think something like `out/Release/node test/paralell/test-*.js` should still give useful output without any tweak to the environment.

In that sense at least this PR needs to also modify test.py to add `NODE_DEBUG=test` to

https://github.com/nodejs/node/blob/f4e4ef5cadf462c38353497c8e16c9cc3b64a5f2/tools/test.py#L197

or even a hint in CONTRIBUTING.md (even though I believe many people don't read it through before contributing) if that's where it's going, otherwise when the test fail, the hint won't be too useful to a new contributor.

Reply:
> If the intention is to get all the tests to slowly refactor to use `common.debuglog`, then I am -1 to this since I think something like `out/Release/node test/paralell/test-*.js` should still give useful output without any tweak to the environment.

@joyeecheung do you mean that it should still give output for tests that fail, or that it should always give output for all tests?

Reply:
@gibfahn I am not sure if it's reasonable to only give output for tests that fail unless we implement this in the test runner - we test our `process.exit()` hooks in JS, among many things. If the hooks are broken, `common.debuglog()` can be broken as well - unless we setup some signal handlers to do this, but I am not sure if that is a good idea.